components/Pusheen.js

```diff
@@ -0,0 +1,46 @@
+const React = require('react');
+
+class Pusheen extends React.Component {
+
+  // TODO: create a componentDidUpdate() here
+  componentDidUpdate() {
+    this.resizePusheen();
+  }
+
+  resizePusheen() {
+    let pusheen = document.getElementById("pusheen");
+    pusheen.height = this.state.size;
+    pusheen.width = this.state.size;
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      size: 100
+    };
+    this.resizePusheen = this.resizePusheen.bind(this);
+  }
+
+  makeBigger() {
+    this.setState({
+      size: this.state.size + 10
+    });
+  }
+
+  makeSmaller() {
+    this.setState({
+      size: this.state.size - 10
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <button onClick={() => this.makeBigger()}>Bigger!</button>
+        <button onClick={() => this.makeSmaller()}>Smaller!</button>
```

Comment:
These two arrow functions can be replaced w/ a simple reference to `this.makeBigger` and `this.makeSmaller` as long as they are bound in the constructor